docs/source/model_design/general_research.rst

```diff
@@ -888,9 +888,125 @@ respiratory disease.
+total number of people in that population at same time.
+
+
+Purpose of study
+~~~~~~~~~~~~~~~~
+
+Cross-sectional studies are used when we aim to find the prevalence of the 
+outcome of interest in a population or subgroups within the population at 
+specific timepoint; or describe a population or subgroups within the population 
+with respect to an outcomes and a set of risk factors. To investigate the 
+relationship between risk factors and the outcome of interest, research often 
+conducted in the form of a survey. Since the study performed at single moment of 
+time point, the limitation is obvious. It cannot show the sequence of events and 
+lack of information for a exposure in the past or future, therefore impossible 
+to infer causality. However, the cross-sectional studies are still useful in 
+designing hypotheses for future research.
+
+
+Types of study
+~~~~~~~~~~~~~~
+
+Cross-sectional studies can be either **descriptive** or **analytical**. 
+Descriptive studies evaluate the outcome or exposure alone, which collects the 
+information on frequency or distribution of health-related variables in a defined 
+population at one time point. (e.g. prevalence of cervical cancer among Chinese 
+women aged 40 to 45) It's a **point prevalence** measure if the data is collected 
+by asking a question "Do you have cervical cancer today?". In contrast, it would 
+become **period prevalence** if question changed to "Have you had cervical cancer 
+on any day within the past month?". Many cross-sectional studies include both 
+questions on current status of the subject and questions about the past, often 
+called retrospective questions, that allow researchers to answer analytical 
+questions from the survey. Analytical studies explore the relationship between 
+exposures (risk factors) and the outcome. The prevalence of the outcome is 
+compared between exposed and unexposed group. It's calculated as prevalence 
+of a disease in exposed population divided by prevalence of that disease in 
+unexposed population, namely the **prevalence ratio**. The prevalence ratio 
+describes the association between the exposure and prevalence of the outcome. 
+For example, we set gender as a risk factor and smoking as the outcome, assuming 
+the prevalence of smoking among men over 15 in 2005 was 30% and 4% for women at 
+same age and year, then the prevalence ratio for smoking is 8. That's being said 
+men are 8 times more likely to be smoker than women in this population, indicating 
+a strong association between sex and prevalence of smoking.
+
+
+Sample selection
+~~~~~~~~~~~~~~~~
+
+Both the method used to select samples and response rate determine how well the 
+results can be generalized to a population as a whole. In a cross-sectional 
+study, the sample is often taken from the whole population or subgroups within 
+the population that are randomly selected and highly representative. It's very 
```

Comment:
If i recall correctly, a lot of the US surveys that reported on their representativeness skewed heavily towards more white populations and more wealthy areas. Because the super-areas were predominantly white anyway, the white sub populations were deemed 'representative' of the super-areas, and we included all of these in our models. This pattern might be worth noting in this section! Not sure how widely it's discussed in other literature?